src/ztc/api/filters.py

```diff
@@ -1,73 +1,122 @@
+from django_filters import rest_framework as filters
 from vng_api_common.filtersets import FilterSet
 
 from ztc.datamodel.models import (
     BesluitType, Eigenschap, InformatieObjectType, ResultaatType, RolType,
     StatusType, ZaakInformatieobjectType, ZaakType
 )
 
+# custom filter to show draft and non-drafts
+PUBLISH_HELP_TEXT = """filter objects depending on their draft status:
```

Comment:
Please rename the `publish` filter to be called `status`. It should still be an enum but accepting

- `alles`: toon objecten waarvan het attribuut `concept` `true` of `false` is.
- `concept`: toon objecten waarvan het attribuut `concept` `true` is.
- `definitief`: toon objecten waarvan het attribuut `concept` `false` is (standaard).